MonoGame.Framework/Platform/Graphics/Texture2D.DirectX.cs

```diff
@@ -230,12 +231,11 @@ protected internal virtual Texture2DDescription GetTexture2DDescription()
 
             return desc;
         }
-
-        internal override void CreateTexture()
+        protected override Resource CreateTexture()
```

Comment:
Keep CreateTexture() as before to return the texture, this will allow us to make _texture private once we rework SwapChainRenderTarget and makes for cleaner code.